Game/imgui.ini

```diff
@@ -5,7 +5,7 @@ Collapsed=0
 
 [Window][ExplorerPanel]
```

Comment:
Out of curiosity, should this ini be added to the gitignore? I noticed it gets generated each time the engine is run, and/or modified. But this also seems client-specific, and not something everyone would share the properties of, if that makes sense. 

Would love a better understanding though!

Reply:
As far as I know, it would be fine adding it to gitignore. It's user specific and has to do with the position of the GUI panels as well as other non critical attributes.

Reply:
if that's the case lets remove the file in this pr and update the .gitignore then

Reply:
removed the file